src/shared/nt-results/ResultsByLoadContainer.react.js

```diff
@@ -97,30 +119,37 @@ class ResultsByLoadContainer extends React.Component {
     })
   }
 
-  renderResults = (version, style, width) => (
-    <div className={styles[`nt__${version}`]} style={{ [style]: `${this.state.x}px` }}>
+  renderResults = (graph, styleName, style) => (
+    <div className={styles[styleName]} style={style}>
       {this.props.data.loading ? (
         <Loading />
       ) : (
         <ResultsByLoad
-          nodes={_.cloneDeep(this.props.data[`graph${version}`].nodes)}
-          links={_.cloneDeep(this.props.data[`graph${version}`].links)}
-          paths={_.cloneDeep(this.props.data[`graph${version}`].paths)}
+          nodes={_.cloneDeep(this.props.data[graph].nodes)}
+          links={_.cloneDeep(this.props.data[graph].links)}
+          paths={_.cloneDeep(this.props.data[graph].paths)}
         />
       )}
     </div>
   )
 
+  renderABGraphs = () => ([
+    this.renderResults('graphA', 'nt__A', { width: `${this.state.x}px` }),
+    <div
+      key='divider'
+      className={styles.nt__divider}
+      style={{ transform: `translateX(${this.state.x}px)` }}
+      ref={c => { this.divider = c }}
+    />,
```

Comment:
- The divider should assign to `this.divider`.
- `renderABGraphs` return an array without unique `key` of each element.